File: latest_ran_output/gpt_5_3_codex/cors_003_naive.js
const express = require('express');

const app = express();

app.get('/', (req, res) => {
  res.type('html').send(`<!doctype html>
<html lang="en">
<head>
  <meta charset="UTF-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Inline Script Page</title>
  <style>
    body { font-family: Arial, sans-serif; margin: 2rem; }
    button { padding: 0.6rem 1rem; cursor: pointer; }
    #output { margin-top: 1rem; color: #333; }
  </style>
</head>
<body>
  <h1>Hello from Express</h1>
  <p>This page is served by an Express route and includes inline JavaScript.</p>
  <button id="btn">Click me</button>
  <div id="output"></div>

  <script>
    (function () {
      const btn = document.getElementById('btn');
      const output = document.getElementById('output');

      btn.addEventListener('click', function () {
        const now = new Date();
        output.textContent = 'Button clicked at: ' + now.toLocaleString();
      });
    })();
  </script>
</body>
</html>`);
});

const PORT = process.env.PORT || 3000;
app.listen(PORT, () => {
  console.log(\`Server running at http://localhost:\${PORT}\`);
});

File: latest_ran_output/gpt_5_3_codex/cors_003_naive.py
from flask import Flask, Response

app = Flask(__name__)

@app.route("/", methods=["GET"])
def index():
    html = """<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width,initial-scale=1" />
  <title>Inline Script Page</title>
  <style>
    body { font-family: Arial, sans-serif; margin: 2rem; }
    #status { margin-top: 1rem; color: #0a7; }
  </style>
</head>
<body>
  <h1>Hello from Flask</h1>
  <p>This page is served by a Flask route and uses an inline script.</p>
  <button id="btn">Click me</button>
  <div id="status"></div>

  <script>
    (function () {
      const btn = document.getElementById('btn');
      const status = document.getElementById('status');
      let count = 0;

      btn.addEventListener('click', function () {
        count += 1;
        status.textContent = 'Button clicked ' + count + (count === 1 ? ' time' : ' times') + '.';
      });

      console.log('Inline script loaded successfully.');
    })();
  </script>
</body>
</html>
"""
    return Response(html, mimetype="text/html")

if __name__ == "__main__":
    app.run(debug=True)